Yeast_cell_growth.ipynb

```diff
@@ -52,7 +52,15 @@
    "source": [
     "library(romero.gateway)\n",
     "library(EBImage)\n",
-    "server <- OMEROServer(host = 'idr.openmicroscopy.org', port = as.integer(IDR_PORT), username = 'IDR_USERNAME', password = 'IDR_PASSWORD')\n",
+    "# Read the configuration\n",
+    "IDR_HOST <- Sys.getenv(\"IDR_HOST\")\n",
+    "IDR_USER <- Sys.getenv(\"IDR_USER\")\n",
+    "IDR_PASSWORD <- Sys.getenv(\"IDR_PASSWORD\")\n",
+    "IDR_PORT <- Sys.getenv(\"IDR_PORT\")\n",
```

Comment:
You can set the default in getenv: https://stat.ethz.ch/R-manual/R-devel/library/base/html/Sys.getenv.html
`Sys.getenv('IDR_PORT', 4064)`

Reply:
good point